actionpack/CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
-*   Fixes multiple calls to `logger.fatal` instead of a single call, 
-    for every line in an exception backtrace, when printing trace 
+*   Fixes Incorrect output from rails routes when using singular resources.
```

Comment:
"Removes excessive whitespace in rails routes..." would better illustrate what the fix entailed :)